charts/kube-prometheus-stack/values.yaml

```diff
@@ -965,8 +965,8 @@ kubeControllerManager:
   ##
   service:
     enabled: true
-    port: 10252
-    targetPort: 10252
+    port: 10257
```

Comment:
I dont like this solution, as it will fail for people who want to update the chart but are still on k8s 1.19.
Imho there should be some logic, using the ports depending on the k8s version or custom ports.

Please also rebase.

Reply:
Sure, adding logic sounds reasonable. So I'll change those defaults to `null` and they will be determined dynamically based on K8s version (helm provides that).

Reply:
Done.

Reply:
Id like to wait for one of the chart maintainers what the think about the solution.